content/applications/productivity/voip/support_calls.rst

```diff
@@ -0,0 +1,126 @@
+=======================
+Support calls with VoIP
+=======================
+
+.. |VOIP| replace:: :abbr:`VoIP (Voice over Internet Protocol)`
+
+A useful tool for support teams, Odoo |VOIP| is used as a way to get in contact with customers that
+need some help. Using |VOIP| can help improve customer satisfaction, reduce support costs, and
+allows support agents to navigate an Odoo database while on a call.
+
+Instead of being tied to their desks, support agents can take calls from anywhere they have access
+to Odoo :doc:`VoIP <../voip>`.
+
```

Comment:
Please move the seealso for the tutorial to the top of this doc. Also, it would be helpful to add a note/important admonition to state that the call queues feature is required in order to proceed– as it may be missed in the first sentence of the next section 
```suggestion

.. important::
   This feature requires at least one :doc:`call queue <axivox/call_queues>` to be configured.

.. seealso::
   `Odoo Tutorial: Working as a Support Agent
   <https://www.odoo.com/slides/slide/working-as-a-support-agent-odoo-voip-4617>`_

```